### PR TITLE
add EWT encoder 

### LINKED ISSUE #106: Add support for EWT addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

We'd like to keep the encoding library as small as possible, so please be efficient in your use of space.

### PR DESCRIPTION
**Pull request by developerfred**
## Issue number

fixed #106 
## Description

- [Wallets use ethers Lib](https://github.com/energywebfoundation/origin/blob/b16289eda5bb41ec93a04ccfc5874b28aab99022/packages/issuer/src/test/Certificate.test.ts)
- [npm/ethers](https://www.npmjs.com/package/ethers)

[explorer energy](https://explorer.energyweb.org/)
https://www.energyweb.org/technology/toolkits/

## List of features added/changed

- 
- 

NOTE: If you are adding new coin address support, please make sure to add a reference link to README so that reviewer can verify.

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by Arachnid** [comment]
I have updated the Travis build to check that entries in the formats, tests, and README lists are in the correct order. Please check your PR and update it to maintain correct ordering in these lists.

**Comment by developerfred** [comment]
Thanks @Arachnid Done!

**Comment by makoto** [comment]
Please specify the reference of where you found the encoding and test data so that I can verify

**Comment by developerfred** [comment]
> Please specify the reference of where you found the encoding and test data so that I can verify
@makoto
- [Wallets use ethers Lib](https://github.com/energywebfoundation/origin/blob/b16289eda5bb41ec93a04ccfc5874b28aab99022/packages/issuer/src/test/Certificate.test.ts)
- [npm/ethers](https://www.npmjs.com/package/ethers)

[explorer energy](https://explorer.energyweb.org/)
https://www.energyweb.org/technology/toolkits/

File: src/index.ts [hunk 1]
```diff
@@ -533,6 +533,7 @@ function icxAddressDecoder(data: string): Buffer {
   }
 }
 
+
```
Comment (makoto) [inline src/index.ts:536]: Please remove extra whitespace

Comment (developerfred) [reply src/index.ts]: done @makoto 0605527

**Pull request merged**